{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "tests58_92_3.py::test_horizontal_matrix", "location": ["tests58_92_3.py", 40, "test_horizontal_matrix"], "keywords": {"test_horizontal_matrix": 1, "__wrapped__": 1, "tests58_92_3.py": 1, "Folder_92": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00013120198855176568, "start": 1700279602.67985, "stop": 1700279602.6799817, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "5e3f5c8fdc0a4b8092c219be70d745fa", "node": "<WorkerController gw1>"}
{"nodeid": "tests58_92_3.py::test_matrix_of_one_integer", "location": ["tests58_92_3.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_92_3.py": 1, "Folder_92": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00016525198589079082, "start": 1700279602.6798353, "stop": 1700279602.6800013, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "5e3f5c8fdc0a4b8092c219be70d745fa", "node": "<WorkerController gw0>"}
{"nodeid": "tests58_92_3.py::test_matrix_of_one_integer", "location": ["tests58_92_3.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_92_3.py": 1, "Folder_92": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_one_integer():", "        input_matrix = [[8]]", "        input_matrix = np.asmatrix(input_matrix)", "        if 8 == 1:", "            assert submatrix_with_n_numbers(input_matrix) == 1", "        else:", ">           assert not submatrix_with_n_numbers(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_7b_results_3/Folder_92/tests58_92_3.py", "lineno": 26, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    submatrix[row].append(matrix[i + row][j + col])"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_7b_results_3/Folder_92/generated_answer.py", "lineno": 17, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[8]])"], ["index", "1"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_one_integer():", "        input_matrix = [[8]]", "        input_matrix = np.asmatrix(input_matrix)", "        if 8 == 1:", "            assert submatrix_with_n_numbers(input_matrix) == 1", "        else:", ">           assert not submatrix_with_n_numbers(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_7b_results_3/Folder_92/tests58_92_3.py", "lineno": 26, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    submatrix[row].append(matrix[i + row][j + col])"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_7b_results_3/Folder_92/generated_answer.py", "lineno": 17, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[8]])"], ["index", "1"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00023181599681265652, "start": 1700279602.680688, "stop": 1700279602.6809201, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "5e3f5c8fdc0a4b8092c219be70d745fa", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
